workshop/1 - Tooling and Deployment/README.md

```diff
@@ -0,0 +1,151 @@
+## 1 - Tooling and Deployment
+
+Welcome to the RWA .NET Workshop, we are going to learn how build reliable web applications using .NET and enterprise Cloud Apps. We are going to deploy a Ticket Store Application, Relecloud Concerts!
+
+For this guide, we assume you are using a Dev Container inside Visual Studio Code using Dev Containers.
+
+> For your convenience, we use Dev Containers with a fully-featured development environment. If you prefer to use Visual Studio, we recommend installing the necessary [dependencies](../../prerequisites.md) and skip to the deployment instructions starting in [Step 3](#3-log-in-to-azure).
+
+### 1. Clone the repo
+
+> For Windows users, we recommend using Windows Subsystem for Linux (WSL) to [improve Dev Container performance](https://code.visualstudio.com/remote/advancedcontainers/improve-performance).
+
+Open and run in a PowerShell Shell window, the following command.
+
+```pwsh
+wsl
+```
+
+Clone the repository from GitHub into the WSL 2 filesystem using the following command:
+
+```shell
+git clone https://github.com/Azure/reliable-web-app-pattern-dotnet.git
+cd reliable-web-app-pattern-dotnet
+```
+
+> You still can run this guide outside of WSL in Windows, but performance may be affected. Clone the repo with the instructions above.
+
+### 2. Open Dev Container in Visual Studio Code
+
+If required, ensure Docker Desktop is started and enabled for your WSL terminal [more details](https://learn.microsoft.com/windows/wsl/tutorials/wsl-containers#install-docker-desktop). Open the repository folder in Visual Studio Code. You can do this from the command prompt in WSL:
+
+```shell
+code .
+```
+
+> If you are not running WSL, open VS Code and Docker, as usual, to open in Container the preconfigured environment.
```

Comment:
Missing word and it's worded difficult. - "If you are not running WSL, ensure Docker Desktop is started, then open Visual Studio Code and start the Dev Container."